doc/releases/migration-guide-3.7.rst

```diff
@@ -381,6 +381,23 @@ Display
             compatible = "sitronix,st7735r";
             reg = <0>;
             spi-max-frequency = <32000000>;
+
+Crypto
+======
+
+* The CSS driver has been deprecated on NXP lpc55s36
+
+Ethernet
+========
+
+* Deprecated NXP's eth_mcux and introduced nxp_enet
```

Comment:
there is already an entry that states "The [nxp,kinetis-ethernet](https://docs.zephyrproject.org/latest/build/dts/api/bindings/ethernet/nxp%2Ckinetis-ethernet.html#std-dtcompatible-nxp-kinetis-ethernet) has been deprecated in favor of [nxp,enet] [...]" -- can't it be piggy-backed?